components/emoji.rst

```diff
@@ -0,0 +1,125 @@
+The Emoji Component
+===================
+
+    The Emoji component provides access to emoji characters and sequences
```

Comment:
Not sure about this part of the description:

```
The Emoji component provides access to emoji characters and sequences...
```

We don't provide tools to get emojis or create emojis ... we could reword it as:

```
The Emoji component provides utilities to work with the emoji characters and
sequences from the `Unicode CLDR dataset`_.
```

Reply:
I will be very transparent here. 

Those files/data beeing here, that felt a bit weird to me to _not_ give methods to get/use them (if not via the Transliterator)..

When i started a PR doing exactly that (inspired by how Intl can give access to countries, or languages), with the goal to release it before 7.1 (so before this documentation would be released).... well it felt pretty clear some had not taken well the emoji/intl split. 

So let's time make its work and i'll retry that later. 

In the meantime i'll rewrite this description :)